api/requirements.txt

```diff
@@ -40,4 +41,4 @@ urllib3==1.26.15
 uvicorn==0.21.1
 uvloop==0.17.0
 watchfiles==0.18.1
-websockets==10.4
+websockets==10.4
```

Comment:
Why do we need websockets ?

Reply:
That was already there, I put the psutil import at the bottom originally and fixing that made it look like I added it